XML/syntax_test_xml.xml

```diff
@@ -201,6 +222,26 @@
      >
 <!-- ^ text - punctuation - illegal -->
 
+     <NS:TAGNAME XMLNS:NS="URI">
+<!-- ^^^^^^^^^^^^^^^^^^^^^^^^^^^ meta.tag -->
+<!--                            ^ - meta.tag -->
+<!-- ^ punctuation.definition.tag.begin -->
+<!--  ^^ entity.name.tag.namespace -->
+<!--    ^ punctuation.separator.namespace -->
+<!--     ^^^^^^^ entity.name.tag.localname -->
+<!--            ^ - entity -->
+<!--             ^^^^^ entity.other.attribute-name.namespace -->
```

Comment:
Just found that, too. XML-namespaces and some attributes must not be matched case insensitive at all. But this is the case prior to my most recent commit, too. 

But honestly - do you want to introduce a pattern matching all maybe allowed or not allowed case-insensitive tags/attributes? 

Me not.